Source/ASDisplayNode.mm

```diff
@@ -3286,6 +3294,21 @@ - (UIView *)preferredFocusedView
 
 @implementation ASDisplayNode (Debugging)
 
++ (void)setShouldStoreUnflattenedLayouts:(BOOL)shouldStore
+{
+  storesUnflattenedLayouts = shouldStore;
```

Comment:
These don't appear to be thread safe. Should we use OSAtomic at least here? I wouldn't be surprised if setting BOOLs *is* actually atomic on most architectures but we probably shouldn't at least have a comment about that or fix it?